daffodil-cli/src/test/scala/org/apache/daffodil/saving/TestCLISaveParser.scala

```diff
@@ -50,7 +48,8 @@ class TestCLISaveParser {
     val shell = Util.start("")
 
     try {
-      String.format("%s save-parser -s %s -r matrix %s", Util.binPath, testSchemaFile, savedParserFile.getName()) !
```

Comment:
These tests used the scala ``!`` to execute the command to save a parser, then used expected to run the command to use the saved parser. We only configure Expect to have the daffodil-lib/src/test stuff on the classpath, so this wouldn't be able to find built-in-formats.xsd. So instead use Expect to execute this command so it's gets the right classpath info.